src/components/SignInForm/SignInForm.js

```diff
@@ -0,0 +1,83 @@
+import { useContext, useState } from "react";
+import UserContext from "../../contexts/UserContext/UserContext";
+import useForm from "../../utils/useForm";
+
+function SignInForm({ switchForm, close }) {
+  const { Api } = useContext(UserContext);
+
+  const [
+    { email, password },
+    { email: emailError, password: passwordError },
+    isValid,
+    onChange,
+  ] = useForm({ inputs: ["email", "password"] });
+
+  const [buttonText, setButtonText] = useState("Sign in");
+  const [formError, setFormError] = useState("");
+
+  function handleSubmit(e) {
+    e.preventDefault();
+
+    setButtonText("Loading...");
+    Api.signIn({ email, password })
+      .then(() => close())
+      .catch((err) => {
+        if (err.status === 401) setFormError("Bad email and/or password");
+        else setFormError("An error occured on the server.");
+      })
+      .finally(() => setButtonText("Sign in"));
+  }
+
+  return (
+    <>
+      <h3 className="user-form__title">Sign in</h3>
```

Comment:
The title is always the same `h3` for any popup with a form. You it should be inside `PopupWithForm` and you need to control it with `title` passing it to the `props`